hooks/tk-nuke/renderer.py

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) 2013 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+import os
+import sys
+import nuke
+
+
+class Renderer(object):
+    def __init__(self):
+        """
+        Construction
+        """
+        self.__app = sgtk.platform.current_bundle()
```

Comment:
Please use self.parent instead of assigning `self.__app`. I know you didn't write this, but we might as well fix it now.